### ISSUE TITLE
macOS Version with Catalyst

### ISSUE THREAD

**Issue by maxkuhlmay** [TortugaPower/BookPlayer#396]
Since Apple introduced Project Catalyst and already developed iPad Apps for macOS, it would be awesome to have a macOS App for BookPlayer.
There is already an iPad Version planned. 
It might be possible to do both versions at the same time.

**Comment by pichfl** [comment]
Thanks for bringing this up and please don't take the following the wrong way, we need to manage expectations. 

While Catalyst is a neat thing, it's also riddled with problems. As with many Apple things, it's best to sit this one out and wait another year or two. This also fits well with the fact there are no resources (neither for design nor development) for an iPad version of BookPlayer as of now. 

Doing both at the same time is a marketing promise from Apple and not a realistic endeavor unless someone has a lot of free time to dump into this. Even so, simply re-using the UI doesn't cut it if we care for something that feels actually native to macOS.

To make the app worthwhile, we also need some way of cross-device syncing. Some work for this was done for the Watch app, but there is more work to be done to make this reliable so people can enjoy their Library seamlessly. 

**tldr** a proper desktop app would be neat, but there is a lot of work that needs to be finished before we even think about introducing a very experimental new technology.

If you want to help us out and speed things along, there are many issues in this project marked as "help wanted", feel free to grab one. If things are unclear, please comment. All help is welcome and very appreciated.

**Comment by skobylite** [comment]
Have you guys had any more thoughts on this since Big Sur? A continuity player would be incredible

**Comment by GianniCarlo** [comment]
We do want to make BookPlayer available wherever possible, but as @pichfl mentioned, we first have to accomplish cross-device synchronization, it's taking some time, but we're working on it. After that, the next major milestone would be to make it available in iPad, then v2 watch app with stand-alone functionality, then enable the app to be on M1 Macs, and see how it behaves 🤔. The priorities may change while we go through things, but those are the major milestones for now, alongside minor feature that we may be adding on the way

**Comment by skobylite** [comment]
Hi team - is cross device sync still on your roadmap?

**Comment by GianniCarlo** [comment]
yup, we're working on it 👍, no estimated date for a beta yet though
